app/services/incoming_text_message_service.rb

```diff
@@ -55,7 +40,7 @@ def self.process(params)
           phone_number: contact_record.from_phone_number,
           body: contact_record.body,
           client: contact_record.client,
-          has_documents: contact_record.documents.present?
+          has_documents: contact_record.documents.present? || params["NumMedia"].to_i > 0,
```

Comment:
not 100% sure about this but i wanted to have a fallback in case the job hadn't finished running and this seems like it would be accurate?

Reply:
(i just confirmed that a job run with `perform_now` will still retry _asynchronously_ if it fails with an error named by `retry_on`), so yeah, we may need to specify here that there are attachments even if documents is blank, and i think this is a reasonable way to do it

Reply:
it looks like the only way this is currently used is to decide to show a link to the client's documents tab in intercom, so this should be fine.